src/Umbraco.Web.UI.Client/src/views/components/content/umb-content-node-info.html

```diff
@@ -19,14 +19,12 @@
                 </ul>
             </umb-box-content>
         </umb-box>
-        <umb-box data-element="node-info-redirects">
+        <umb-box data-element="node-info-redirects" style="display:none;" ng-cloak ng-show="!urlTrackerDisabled && hasRedirects">
```

Comment:
@marcemarc I'm (finally) using Redirects in vain, (my first 'real' project on v8) but I couldn't get the redirects panel to show on the Info app. I've pinpointed it down to the combination of `ng-show` and inline `display:none` - doesn't work together.

I swapped out the `style="display:none;" ng-cloak ng-show` with an `ng-if`, which works in my instance.

Before I start preparing a PR for this, I'm curious, did this work for you? (or anyone else?)

Reply:
Hi @leekelleher this was added to V7 and worked at the time... and I've just been reminded to reply here, as I've just seen it on a V7 site we are currently working on! I'm guessing something got changed in V8 or in the Angular version change... I don't profess to understand why or how it did work... but it did work and still does! on V7! - but it's kinda hidden functionality so I doubt it would have been on anyone's radar to test.... I sort of think though it shouldn't be doing this show and hide boxy thing that only appears if redirects are present, and if I was creating it now, I'd perhaps implement the 'OtherUrls' function of a custom UrlProvider, to make the redirect Urls appear in the same Urls box as the main Url... but maybe it is better to separate it I don't know!

Reply:
Follow up on this (if anyone sees this in future), this has been resolved for v8, in PR #8713.